### ISSUE TITLE
[XMPP] can't connect

### ISSUE THREAD

**Issue by sotitrox** [loqui/im#1097]
I can connect after that upgrade that includes the bosh url(long time ago...), only stay on "connecting...".

This is my server:

**inbox.im**
host: **xmpp.inbox.im**(if default connection method don't work)
source: http://inbox.im/setup

Please, help me.
Thanks.

PD= my others accounts in others servers also don't work with loqui.

**Comment by cmeerw** [comment]
Have you tried connecting via tcp:xmpp.inbox.im:5222?

**Comment by sotitrox** [comment]
yeah, it's working :), but where is the documentation for this?

**Comment by ghost** [comment]
Not work for me, can't connect with any XMPP

**Comment by nfsprodriver** [comment]
Which version do you use? The host address must be sth. like https://server.com/http-bind/ (Bosh). Not all XMPP server support bosh using http-bind

**Comment by ghost** [comment]
I try with suchat.org, riseup.net and jabjab.de and not works, the "Connect via" should be auto-discovered

**Comment by nfsprodriver** [comment]
Yes, but tcp and so on doesn't seem to work anymore. Since v0.5.23 there's a new autocorrection to such above. Or you enter it manually. If you still can't login, the server doesn't support it.

**Comment by ghost** [comment]
I'm using v0.5.23, not work with my accounts, is hard for the user try to divine the bosh url

**Comment by nfsprodriver** [comment]
Sorry for you. jabb.im and the hot-chilli servers work for me. Anyway I was thinking about removing XMPP from loqui, but indeed some people use it. So I'll keep it, but my focus is on WhatsApp.

**Comment by ghost** [comment]
Hooo bad, really sad :/

**Comment by nfsprodriver** [comment]
Currently we have too less active developers for this app (me and one or two others). I joined the project about a year ago with no knowledge. XMPP is a part I didn't touch much and for deeper work I haven't got enough time. Maybe some others can help you with that :)
